src/Pages/VolunteerPage.jsx

```diff
@@ -0,0 +1,153 @@
+                  className={classes.mainImage}
+                />
+              </Box>
+            </Grid>
+            <Grid item md={6}>
+              <Box justifyContent="center" display="flex">
+                <Box className={classes.contentBox}>
+                  <Typography className={classes.textColor} variant="h2">
+                    Why Volunteer?
+                  </Typography>
+                  <Typography className={classes.textColor} variant="body1">
+                    Conviction vacation improves the chances of successful
+                    reentry into society for formerly incarcerated individuals
+                    and reduces recidivism.
+                  </Typography>
+                  <Typography className={classes.textColor} variant="body1">
+                    By volunteering, you will help us reduce barriers and
+                    streamline the process of vacating eligible convictions in
+                    Washington state to make it easier for people to move
+                    forward.
+                  </Typography>
+                </Box>
+              </Box>
+            </Grid>
+          </Grid>
+        </Box>
+        <Grid
+          container
+          justifyContent="center"
+          alignItems="center"
+          style={{ marginTop: "0rem" }}
+        >
+          <Grid item md={6} className={classes.backgroundLightSecondary}>
+            <Box className={classes.contentBox}>
+              <Typography variant="h2" className={classes.textColor}>
+                Who can apply?
+              </Typography>
+              <BulletedListMui>
+                <li>You can commit to 4 hours a week for at least 6 months</li>
+                <li>
+                  You want to share your expertise in a field to help us make a
+                  better product
+                </li>
+                <li>
+                  You are self-motivated and are comfortable with remote work
+                </li>
+              </BulletedListMui>
+            </Box>
+          </Grid>
+          <Grid item md={6} className={classes.backgroundWhite}>
+            <Box className={classes.contentBox}>
+              <Typography className={classes.textAltColor} variant="h2">
+                How can you apply?
+              </Typography>
+              <Typography variant="body1">
+                We are currently using DemocracyLab to streamline the
+                volunteering process, which you can find clicking the button
+                below.
+              </Typography>
+              <Typography variant="body1">
+                If you want further information and want to have a chat, don’t
+                hesitate to contact us though our form.
+              </Typography>
+            </Box>
+          </Grid>
+        </Grid>
+        <Box className={classes.backgroundSecondary}>
+          <Typography className={classes.rolesHeading} variant="h2">
+            Open Roles
```

Comment:
Can we add or adjust the padding here? 
<img width="517" alt="Screen Shot 2022-01-12 at 11 45 36 AM" src="https://user-images.githubusercontent.com/25576005/149194061-acf383f8-fa20-4859-aabc-2b7e686689c4.png">

Reply:
Definitely.

Reply:
![image](https://user-images.githubusercontent.com/15237338/149196991-b5ec9f5f-d261-45a5-8dc1-6389a257db7a.png)
How is this?